global.d.ts

```diff
@@ -5,7 +5,7 @@ import { GurpsCombatant } from './module/combat/combatant.ts'
 import { GurpsItem } from './module/item.js'
 import { GurpsToken } from './module/token/gurps-token.ts'
 
-export { }
+export {  }
```

Comment:
not an issue, but why is there seemingly another space added here with every PR?

Reply:
No idea. Something with my prettier config?